tsdb/config.go

```diff
@@ -128,3 +130,24 @@ func NewConfig() Config {
 		QueryLogEnabled: true,
 	}
 }
+
+func (c *Config) Validate() error {
```

Comment:
This pattern, of adding `Validate` to individual config types, exists elsewhere in the code.